components/input/Input.js

```diff
@@ -41,6 +48,9 @@ export default class Input extends Component {
     inverse: false,
     disabled: false,
     placeholder: '',
+    precision: null,
+    min: -9007199254740991,
+    max: 9007199254740991,
```

Comment:
What the actual fuck. Do we really need these here, and why not use `Number.MAX_SAFE_INTEGER`/`Number.MIN_SAFE_INTEGER` if necessary?